src/main.zig

```diff
@@ -573,6 +573,10 @@ pub fn main() void { // MARK: main()
 	defer server.terrain.globalDeinit();
 
 	if(headless) {
+		server.world_zig.tryCreateWorld(settings.launchConfig.autoEnterWorld, settings.launchConfig.worldConfig) catch |err| {
```

Comment:
Doesn't this mean that it recreates the world every time, likely overwriting existing world settings (e.g. game rules) in the process.

Please check if the world exists first.

Reply:
https://github.com/PixelGuys/Cubyz/blob/8df1b04a29718068e4be1dacf5254ad41b5800fe/src/server/world.zig#L68

I thought it would fail to create the world if it already existed, but I can add some logic to check that it does before attempting to create it.